scipy/stats/tests/test_distributions.py

```diff
@@ -4112,3 +4112,18 @@ def test_loguniform():
     vals, _ = np.histogram(np.log10(rvs), bins=10)
     assert 900 <= vals.min() <= vals.max() <= 1100
     assert np.abs(np.median(vals) - 1000) <= 10
+
+
+def test_argus_rvs_large_chi():
+    np.random.seed(325)
+    # test that the algorithm can handle large values of chi
+    x = stats.argus.rvs(50, size=500)
```

Comment:
instead of setting the random seed above could we do `x = stats.argus.rvs(50, size=500, random_state=325)`, and same in the next test?

Reply:
yes, that makes sense. I changed it